substrate/frame/identity/src/types.rs

```diff
@@ -319,6 +324,29 @@ pub struct RegistrarInfo<
 	pub fields: IdField,
 }
 
+/// The number of usernames that an authority may allocate.
+type Allocation = u32;
+/// The maximum allowed length of a suffix.
+const SUFFIX_MAX_LENGTH: u32 = 7;
```

Comment:
Can this also be configurable? There is no justification in the docs of why this is a good value.